main.asm

```diff
@@ -70,22 +70,22 @@ _start:
 	; Register signal handlers ( just close all other threads by jumping to SYS_EXIT_GROUP )
 	mov r10, 8 ; sizeof(sigset_t) displays 128, but strace shows 8 ... so 8 it is! -_-
 	xor rdx, rdx
-	mov QWORD [sa_handler], exit
-	mov rsi, sigaction
+  ;mov QWORD [rel sa_handler], exit
```

Comment:
pretty sure we don't want to comment this out, but I could not figure out how to fix this one line.  If I understand correctly, we want to move a quad word (64b == 8B), the address of label exit into the memory block labeled sa_handler?  Not sure how to avoid the warning:
```
main.asm:73: error: macho: sorry, cannot apply 32 bit absolute relocations in 64 bit mode, consider "[_symbol wrt rip]" for mem access, "qword" and "dq _foo" for pointers.
```